docs/developers.rst

```diff
@@ -29,4 +29,4 @@ Tests
 Test for the module compare and its functions
 *********************************************
 
-.. literalinclude:: ../tests/test_class.py
+.. literalinclude:: ../tests/test_compare.py
```

Comment:
I wouldn't recommend this in larger projects (you have to manually edit the line numbers every time, and other changes). Check the sphinx extension [autodoc](https://docs-python2readthedocs.readthedocs.io/en/master/code-doc.html) if you want to know how it's done automatically.